tests/fakes.py

```diff
@@ -123,14 +123,41 @@
 LIVEPATCH_CHECK_FAILED = LIVEPATCH_ENABLED_STATUS.format(
     check_state='check-failed', patch_state='does not matter')
 
-UA_STATE_ELSEWHERE = """
-cat <<EOF
-fips: disabled
-esm: enabled
+STATUS_CACHE_LIVEPATCH_ENABLED = """esm: disabled (not available)
```

Comment:
please use `"""\` on the first line so you can keep the text aligned (same below)